com.sap.cloud.lm.sl.cf.core/src/main/resources/com/sap/cloud/lm/sl/cf/core/db/changelog/db-changelog-add_quartz_tables.xml

```diff
@@ -0,0 +1,354 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I believe it is standard - get from here:
http://www.utilities-online.info/articles/Quartz-2.2.3-Liquibase-table-generation/quartz-changelog.xml

The only difference is removed onUpdate="NO ACTION" because it is not supported for Hana and it is the default behavior.

Reply:
Good.